offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsController.groovy

```diff
@@ -68,6 +66,27 @@ class MaintainProductsController {
         }
     }
 
+    /**
+     * Triggers the copy Use Case of a product
```

Comment:
```suggestion
     * Triggers the copy use case of a product
```

Reply:
Capital Case Makes Everything More ReadAble And Is AweSome